Build gradle error when run any androidTest

**Summary:** 

when i run any test in androidTest it throw build an error
**Steps to reproduce:** 
Open androidTest file
run any androidTest file

**System logs:**

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkProdDebugAndroidTestDuplicateClasses'.
> 1 exception was raised by workers:
  java.lang.RuntimeException: java.lang.RuntimeException: Duplicate class org.hamcrest.BaseDescription found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.BaseMatcher found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.CoreMatchers found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.Description found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.Factory found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.Matcher found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.SelfDescribing found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.StringDescription found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.AllOf found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.AnyOf found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.DescribedAs found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.Is found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.IsAnything found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.IsEqual found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.IsInstanceOf found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.IsNot found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.IsNull found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.core.IsSame found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.internal.ArrayIterator found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.internal.SelfDescribingValue found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  Duplicate class org.hamcrest.internal.SelfDescribingValueIterator found in modules jetified-hamcrest-core-1.3.jar (org.hamcrest:hamcrest-core:1.3) and jetified-mockito-all-1.10.8.jar (org.mockito:mockito-all:1.10.8)
  
  Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.


* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

```

**Device and Android version:** 
Android Device :Redmi y3(API :29)
**Commons app version:** 
latest master

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -97,12 +97,6 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.3.1"
     testImplementation 'com.facebook.soloader:soloader:0.9.0'
 
-    // Mockito and PowerMock
-    androidTestCompile ('org.powermock:powermock-mockito-release-full:1.6.0') {
-        exclude module: 'hamcrest-core'
-        exclude module: 'objenesis'
-    }
-
     // Android testing
     androidTestImplementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'

